update pre-req to include min service quotas (#338)
astronomer/docs: cloud/install-aws.md

Before completing this setup, make sure that you have:

- A dedicated AWS account with minimum EC2 service quotas
- A user that has `CreateRole` permissions on that account
- Subscribed to the [Astronomer Cloud Status Page](https://cloud-status.astronomer.io/). This will ensure that you're alerted in the case of an incident or scheduled maintenance.

> Astronomer Cloud requires a dedicated AWS account, with minimum required EC2 service quotas to mitigate near term capacity issues. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, follow [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/). If you need to modify or increase a specific quota, see Amazon’s documentation on [requesting a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html).  Once your AWS account is created, proceed to Step 1.
> 
> | QuotaCode  | QuotaName                                                        | Minimum Required Value  |
> | -----------| ---------------------------------------------------------------- | ----------------------- |
> | L-1216C47A | Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances | 40                      |
> | L-43DA4232 | Running On-Demand High Memory instances                          | 40                      |
> | L-34B43A08 | All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Requests  | 40                      |

Reviewer: ```suggestion
Astronomer Cloud requires a dedicated AWS account with a minimum set of EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, follow [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/). 

The required EC2 service quotas are:

| QuotaCode  | QuotaName                                                        | Minimum Required Value  |
| -----------| ---------------------------------------------------------------- | ----------------------- |
| L-1216C47A | Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances | 40                      |
| L-43DA4232 | Running On-Demand High Memory instances                          | 40                      |
| L-34B43A08 | All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Requests  | 40                      |

These are required to mitigate near term capacity risks and ensure a smooth onboarding experience on Astronomer Cloud. If you need to modify or increase a specific quota, see Amazon’s documentation on [requesting a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html). 

Once your AWS account is created, proceed to Step 1.
```

Reviewer: @chomipi88 Does the customer have to set these when they create the AWS account? I.e. we don't do this for them, yes?

Before completing this setup, make sure that you have:

> - A dedicated AWS account with minimum EC2 service quotas
> - A user that has `CreateRole` permissions on that account
> - Subscribed to the [Astronomer Cloud Status Page](https://cloud-status.astronomer.io/). This will ensure that you're alerted in the case of an incident or scheduled maintenance.

Reviewer 2: ```suggestion
- A dedicated AWS account with minimum EC2 service quotas.
- A user that has `CreateRole` permissions on that account.
- Subscribed to the [Astronomer Cloud Status Page](https://cloud-status.astronomer.io/). This will ensure that you're alerted in the case of an incident or scheduled maintenance.
```

> Astronomer Cloud requires a dedicated AWS account with a minimum set of EC2 service quotas. For security reasons, the install process is not currently supported on an AWS account that has other tooling running in it. For instructions on creating a new AWS account, follow [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/). 

Reviewer 2: ```suggestion
For security reasons, Astronomer Cloud requires a fully dedicated AWS account with a minimum set of EC2 service quotas. To create a new AWS account, follow [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/). 
```

Reviewer 2: Thought second sentence was duplicative, as we are just explaining what a dedicated account is in a different way.

> The required [EC2 service quotas](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-resource-limits.html) are:

Reviewer 2: ```suggestion
The following table lists all required [EC2 service quotas](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-resource-limits.html) for Astronomer Cloud:
```

Reviewer 2: In the previous paragraph, we say that a dedicated account with quotas is required for security purposes. But here we say that the quotas help with onboarding/ capacity. Should we remove the mention of quotas in the previous paragraph and focus on their benefit here instead?

| QuotaCode  | QuotaName                                                        | Minimum Required Value  |
| -----------| ---------------------------------------------------------------- | ----------------------- |
| L-1216C47A | Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances | 40                      |
| L-43DA4232 | Running On-Demand High Memory instances                          | 40                      |
| L-34B43A08 | All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Requests  | 40                      |

> These are required to mitigate near term capacity risks and ensure a smooth onboarding experience on Astronomer Cloud. If you need to modify or increase a specific quota, see Amazon’s documentation on [requesting a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html). 

Reviewer 2: In the previous paragraph, we say that a dedicated account with quotas is required for security purposes. But here we say that the quotas help with onboarding/ capacity. Should we remove the mention of quotas in the previous paragraph and focus on their benefit here instead?

> These are required to mitigate near term capacity risks and ensure a smooth onboarding experience on Astronomer Cloud. If you need to modify or increase a specific quota, see Amazon’s documentation on [requesting a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html). 

Reviewer 2: ```suggestion
If you need to modify or increase a specific quota, see Amazon’s documentation on [requesting a quota increase](https://docs.aws.amazon.com/servicequotas/latest/userguide/request-quota-increase.html). 
```
